ods_ci/tests/Resources/Page/Operators/ISVs.resource

```diff
@@ -32,6 +32,11 @@ Install ISV Operator From OperatorHub Via CLI    # robocop: disable
     ${rc}    ${out}=    Run And Return Rc And Output    sed -i'' -e "s/<CATALOG_SOURCE>/${catalog_source_name}/g" ${operator_sub_filepath}    # robocop: disable
     ${rc}    ${out}=    Run And Return Rc And Output    sed -i'' -e "s/<CS_NAMESPACE>/${cs_namespace}/g" ${operator_sub_filepath}    # robocop: disable
     Oc Apply    kind=Subscription    src=${operator_sub_filepath}
+    Sleep    60s    reason=Waiting extra time to make sure the installplan is available
```

Comment:
what if the upgade approval is not set to manual? We don't need to do this in that case. So it would be bettet to check the update approval first no?

Reply:
yeah, changed the logic to just retrieve the installplan from the subscription and approve only if the approval is setted to manual.

Thanks @bdattoma for the review!